pkgs/applications/graphics/djv/default.nix

```diff
@@ -111,6 +111,11 @@ stdenv.mkDerivation rec {
   patches = [
     # Pull fix ending upstream inclusion for gcc-12+ support:
     #   https://github.com/darbyjohnston/DJV/pull/477
+    (fetchpatch {
```

Comment:
the above comment is not for this patch

Reply:
The comment above actually applies to all 3 patches that were already listed, and the new one is the only commit that was left out in that list: https://github.com/darbyjohnston/DJV/pull/477/commits

My guess is that it wasn't included, because it wasn't strictly necessary to build DJV *yet* (before GCC 13).

Would you want to include a comment for each one of them instead? Or should we fetch the entire PR as a patch instead?

(Since it has been merged, this link should be stable: https://github.com/darbyjohnston/DJV/pull/477.patch)

Reply:
Thank you for your explanation. Leave it as is for now.